ibmcloud/terraform/cluster/ansible/playbook.yml

```diff
@@ -308,7 +311,7 @@
       shell: |
         set -o errexit -o pipefail
         cd /root
-        git clone -b peerpod-2022.04.04 https://github.com/yoheiueda/cloud-api-adaptor.git
+        git clone -b zvsi https://github.com/liudalibj/cloud-api-adaptor.git
```

Comment:
@liudalibj, @yoheiueda - Hi Da Li & Yohei, I wonder what the plan is once this PR merges.  Would we consider creating a new tag and then update this playbook to pull that version, or we could keep Da Li's branch until we are ready to contribute the whole `code-import` branch in Yohei's fork to `staging` (which might be around the end of the week)?